src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -182,35 +182,32 @@ function bindCopyButton() {
         return false;
     });
 }
+var prevRow;
```

Comment:
A comment like 'Variable is intended to be outside of bindCopyEvents function scope, it is used to remember what row was previously clicked by the user' will suffice.

Note that the previous working version did not have such a comment so it was prone to being removed as it was probably thought of as an error or something that could not be understood.

Perhaps, instead of abbreviating it as prevRow, it can be named as $previouslySelectedCopyRow, with that it will be immediately obvious that it is a jQuery wrapped object and that is relates to the bindCopyEvents function below it.